addons/udes_security/static/src/js/mail/utils.js

```diff
@@ -0,0 +1,30 @@
+odoo.define("udes_security.mail_utils", function (require) {
+  "use strict";
+
+  var mail_utils = require("mail.utils");
+
+  function udes_addLink(node, transformChildren) {
+    if (node.nodeType === 3) {  // text node
+	/* Don't add automatic link to url text
```

Comment:
Is this comment/commented code needed?

Reply:
Yes, absolutely! Otherwise odoo will add back link to the plain text urls when displaying in the UI/frontend.